main.go

```diff
@@ -413,6 +450,51 @@ func convert(req *plugin.CodeGeneratorRequest) (*plugin.CodeGeneratorResponse, e
 	return res, nil
 }
 
+func addExtensions(msg *descriptor.DescriptorProto, extensions []*descriptor.FieldDescriptorProto, pkg string) {
+	for _, ext := range extensions {
+		extendeeNodes := strings.Split(ext.GetExtendee(), ".")
+		pkgNodes := strings.Split(pkg, ".")
+
+		// We already know this extendee cannot be in this package.
+		if len(pkgNodes) > len(extendeeNodes) {
+			continue
+		}
+
+		// skipping leading "."
+		nodeIdx := 1
+		for _, node := range pkgNodes {
+			if node != extendeeNodes[nodeIdx] {
+				nodeIdx = 0
+				break
+			}
+			nodeIdx += 1
```

Comment:
:older_woman: 

```suggestion
			nodeIdx++
```

Reply:
What, I didn't know we could do that in Golang... Is that a new feature ?

Reply:
:smile: No. You should setup https://golangci-lint.run/usage/install/#local-installation there is a lot to learn from it

Reply:
Installed